### PR TITLE
feat(helm): cleanup chart unused components

### PR DESCRIPTION
**Pull request by alpiquero**
## Description
Removed unused components from the chart
* Mongo-writer
* Mongo-express
* InfluxDB
* Kapacitor

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Other changes (ci configuration, documentation or any other kind of changes)

## Checklist:

- [ ] I have created tests for my code changes, and the tests are passing.
- [X] I have executed the pre-commit hooks locally.
- [ ] My change requires a change to the documentation (create a new issue if the documentation has not been updated).
- [X] I have updated the documentation accordingly.

### CODE REVIEW

File: helm/kai/CHART.md [hunk 1]
```diff
@@ -5,8 +5,6 @@
 | Repository | Name | Version |
 |------------|------|---------|
 | https://charts.min.io/ | minio | 5.0.14 |
-| https://helm.influxdata.com/ | influxdb | 4.8.1 |
```
Comment (dfinteligenz) [inline helm/kai/CHART.md:8]: The influx repository is still being used in KAI, we need to remove all references before removing the instance from the chart.

**Review by dfinteligenz — CHANGES_REQUESTED** [review]
We need to maintain the Influx until we do a refactor on Kai

**Review by dfinteligenz — CHANGES_REQUESTED** [review]
We need to maintain the Influx instance until we do the refactor on KAI

**Comment by alpiquero** [comment]
@dfinteligenz  we will block this PR until API refactors had been completed on this regard

**Comment by dfinteligenz** [comment]
> @

Ok, set it as Draft then please.

**Pull request closed without merging**